e2e_tests/src/objects/common.js

```diff
@@ -281,8 +265,8 @@ class Common {
         await this.fillAddressDetails();
         await this.fillTermsAndConditions();
         await this.waitForAccountDropdown();
-        await this.removeLoginState(VIRTUAL_SIGNUP_STATE);
-        await qawolf.saveState(this.page, REAL_SIGNUP_STATE);
+        // await this.removeLoginState('VIRTUAL');
+        // await this.saveState('REAL');
```

Comment:
If we need to keep these comments, please add a TODO for future reference

Reply:
Ah! We don't. Nice catch @mahdiaryayi-fs .